apps/server/src/modules/user-import/uc/user-import.uc.ts

```diff
@@ -185,6 +186,7 @@ export class UserImportUc {
 			throw new BadRequestException('School cannot exit from user migration mode');
 		}
 		school.inUserMigration = false;
+		school.inMaintenanceSince = undefined;
```

Comment:
not sure if want this...on one hand, since we have problems and the timeout, it makes sense to finish it all with a single action.
But, in the frontend/UI, after the users are saved, the end of maintanence is separated step. It has its own endpoint below endSchoolInMaintenance. And there was some reasoning behind it...

Reply:
This method gets only called after the migration is finished in the migrate step. For now it should be ok. But yes, when we rebuild all this we need to extract this and make it all safer